src/app/cases/styles.tsx

```diff
@@ -14,25 +8,37 @@ export const FiltersContainer = styled.div`
   gap: 1rem;
 `;
 
-export const MainDisplay = styled.main`
+export const PageContainer = styled.div`
+  height: 100vh;
+  width: 100vw;
   display: grid;
-  grid-template-columns: 5fr 10fr;
-  margin-top: 2rem;
-  width: 100%;
+  grid-template-columns: 1fr;
+  grid-template-rows: 0.2fr 1fr;
+  /* margin: 2rem; */
+  padding: 1rem 2rem 0rem 2rem;
+  overflow: hidden;
 `;
 
-// cards
 export const CardColumn = styled.div`
-  display: flex;
+  overflow-y: scroll;
   flex-direction: column;
-  gap: 2rem;
-  border-right: 1px solid black;
-  padding-top: 3px;
-  padding-right: 2rem;
+  column-gap: 2rem;
+  justify-content: space-between;
+  border-right: 1px solid ${COLORS.greyLight};
+  margin-top: 1.5rem;
+  padding-right: 3rem;
```

Comment:
this padding-right gets overridden by the following padding (i think), so you can remove this line

Reply:
oops, forgot to remove that!